doc/autoyast.md

```diff
@@ -170,29 +203,42 @@ Note that there are no separate classes for {Y2Storage::Planned::Disk} and
 in the {Y2Storage::Proposal::AutoinstDevicesCreator} for historical reasons and,
 ideally, it should be extracted.
 
+It is worth to mention that, if not enough space is found, AutoYaST will try
+to reduce the size of the planned devices to make them fit. Check the
+{Y2Storage::Proposal::AutoinstPartitionSize} for further details.
+
+### Phase six: adding boot devices (if needed)
+
+After adjusting the devicegraph to include the list of planned devices, AutoYaST
+checks whether the resulting system is bootable. If that's not the case,
+AutoYaST will try to add the needed devices to the devicegraph.
+
+{Y2Storage::BootRequirementsChecker} is the class responsible for finding out
+whether the system is bootable and determining the missing partitions (if any).
+
 ## Issues reporting
 
 It is possible that, given a profile, AutoYaST finds issues when trying to
-figure out a the partitioning layout. Some of those issues might be serious
-enough to stop the installation; in other cases, just displaying a warning could
-be the way to go.
+figure out the partitioning layout. Some of those issues might be serious
+enough to stop the installation; in other cases, just displaying a warning is
+the way to go.
```

Comment:
maybe I would simplify it to `Some of those issues might be serious
￼enough to stop the installation; in other cases, it just displays a warning.`